cmd/cli/main.go

```diff
@@ -101,6 +101,23 @@ func main() {
 		},
 	}
 
+	cmdRefresh := &cobra.Command{
+		Use:   "refresh [cluster id]",
+		Short: "delete api key associated with [cluster id] and genereate a new key",
+		Args:  cobra.MinimumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			key, err := smtpdetailsClient.Refresh(args[0])
+			if err != nil {
+				if smtpdetails.IsNotExistError(err) {
+					//message appropriate?
+					exitError(fmt.Sprintf("api key for cluster %s does not exist", args[0]), exitCodeErrKnown)
```

Comment:
Maybe something like:

```
cannot create api key for cluster that does not exist, cluster=%s, use the create command
```

Something like that, to indicate they've never ran `./cli create <clusterId>` before and they should do that instead.

Reply:
Sounds great